test/system/freertos/main.c

```diff
@@ -33,7 +33,7 @@ int main(void)
 
 	/* Will not get here, unless a call is made to vTaskEndScheduler() */
 	while (1) {
-		__asm__("wfi\n\t");
```

Comment:
This file also needs a '#include "metal/compiler.h" to get around an implicit declaration warning.

Reply:
Thanks! Fixed in 2c83792487bca1e50fea2bdf7c52ccfd5608505f 👍